### PR TITLE
nodejs-ca: Add docs file - index.md

### PR DESCRIPTION
**Pull request by Tsuna-mi**
This PR is related to [#1197](https://github.com/nodejs/nodejs.org/issues/1197)

I have created the '**Docs**' index.md translation to catalan, according Node.js Website Translation Policy indications.

nodejs-ca: Add docs file - index.md

### CODE REVIEW

File: locale/ca/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+La [Referència de l'API](/api/) proporciona informació detallada sobre una funció ó un objecte en Node.js. Aquesta
+documentació indica quins arguments acepta un mètode, el valor que retorna aquest mètode i quins errors poden estar
+relacionats al mateix. També indica quins mètodes estàn a l'abast per les diferents versions de Node.js
+
+També descriu els mòduls inclosos que proporciona Node.js, mes no documenta els mòduls que proporciona la comunitat.
+
+<div class="highlight-box">
+    <h4>¿Buscant la referència de l'API per a una versió anterior?</h4>
+
+    <ul>
```
Comment (fhemberger) [inline locale/ca/docs/index.md]: Please update:

```html
    <ul>
        <li><a href="https://nodejs.org/docs/latest-v7.x/api/">Node.js 7.x</a></li>
        <li><a href="https://nodejs.org/docs/latest-v5.x/api/">Node.js 5.x</a></li>
        <li><a href="https://nodejs.org/docs/latest-v4.x/api/">Node.js 4.x</a></li>
        <li><a href="https://nodejs.org/docs/latest-v0.12.x/api/">Node.js 0.12.x</a></li>
        <li><a href="https://nodejs.org/docs/latest-v0.10.x/api/">Node.js 0.10.x</a></li>
        <li><a href="https://nodejs.org/docs/">all versions</a></li>
    </ul>
```

Comment (Tsuna-mi) [reply locale/ca/docs/index.md]: Done! Thanks @fhemberger

File: locale/ca/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        <li><a href="https://nodejs.org/docs/latest-v4.x/api/">Node.js 4.x</a></li>
+        <li><a href="https://nodejs.org/docs/latest-v0.12.x/api/">Node.js 0.12.x</a></li>
+        <li><a href="https://nodejs.org/docs/latest-v0.10.x/api/">Node.js 0.10.x</a></li>
+        <li><a href="https://nodejs.org/docs/">Totes les versions</a></li>
+    </ul>
+</div>
+
+### Característiques d'ES6
+
+La [secció d'ES6](/en/docs/es6/) descriu l'arbre dels grups de les característiques d'ÉS6, i detalla quines
```
Comment (jose-lara) [inline locale/ca/docs/index.md]: Remove the accent mark at "ÉS6"

File: locale/ca/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+característiques sont activades per defecte en Node.js, juntament amb enllaços explicatius. També mostra com trobar
```
Comment (jose-lara) [inline locale/ca/docs/index.md]: "són" instead of "sonst"

File: locale/ca/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+què versió de V8 fa servir una versió particular de Node.js.
```
Comment (jose-lara) [inline locale/ca/docs/index.md]: "quina" instead of "què"

File: locale/ca/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+labels:
+  lts: LTS
+---
+
+# Sobre la documentació
+
+Existeixen tres tipus de documentació a l'abast en aquest lloc:
+
+* Referència de la API
+* Característiues d'ES6
```
Comment (jose-lara) [inline locale/ca/docs/index.md]: typo, forgot a "q" in "Característiues"

File: locale/ca/docs/index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+### Referència de l'API
+
+La [Referència de l'API](/api/) proporciona informació detallada sobre una funció ó un objecte en Node.js. Aquesta
+documentació indica quins arguments acepta un mètode, el valor que retorna aquest mètode i quins errors poden estar
+relacionats al mateix. També indica quins mètodes estàn a l'abast per les diferents versions de Node.js
+
+També descriu els mòduls inclosos que proporciona Node.js, mes no documenta els mòduls que proporciona la comunitat.
+
+<div class="highlight-box">
+    <h4>¿Buscant la referència de l'API per a una versió anterior?</h4>
```
Comment (jose-lara) [inline locale/ca/docs/index.md]: In catalan, there is only one question mark at the end of the sentence :)

Comment (Tsuna-mi) [reply locale/ca/docs/index.md]: Thanks for review it @jose-lara!
I've change it all your proposals 💯

File: locale/ca/docs/index.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+* Referència de la API
+* Característiques d'ES6
+* Preguntes freqüents
+
+### Referència de l'API
+
+La [Referència de l'API](/api/) proporciona informació detallada sobre una funció ó un objecte en Node.js. Aquesta
+documentació indica quins arguments acepta un mètode, el valor que retorna aquest mètode i quins errors poden estar
+relacionats al mateix. També indica quins mètodes estàn a l'abast per les diferents versions de Node.js
```
Comment (jose-lara) [inline locale/ca/docs/index.md]: están, is without accent mark. In fact, I think it could be better if you write here "són"

Comment (Tsuna-mi) [reply locale/ca/docs/index.md]: Oki doki :D

**Review by jose-lara — APPROVED** [review]

**Comment by fhemberger** [comment]
Great, thank you!

**Pull request merged**